Client/Forms/FrmRemoteChat.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Windows.Forms;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+using System.Threading;
+using xClient.Core.Helper;
+using xClient.Core.Networking;
+using xClient.Core.Utilities;
+
+namespace xClient.Forms
+{
+    public partial class FrmRemoteChat : Form
+    {
+        private Client _connectedClient;
+
+        public FrmRemoteChat(Client client)
+        {
+            this._connectedClient = client;
+            InitializeComponent();
+
+        }
+
+        public void AddMessage(string sender, string message)
+        {
+            txtMessages.AppendText(string.Format("{0} {1}: {2}{3}", DateTime.Now.ToString("HH:mm:ss"), sender, message, Environment.NewLine));
+        }
+
+        private void FrmRemoteChat_FormClosing(object sender, FormClosingEventArgs e)
+        {
+            e.Cancel = true;
+        }
+
+        private void StartKeepingTopMost()
+        {
+            (new Thread(() => 
+            {
+                while (Visible)
+                {
+                    NativeMethods.SetForegroundWindow(Handle);
+                    Thread.Sleep(1);
```

Comment:
This seems a bit too low. Sleeping the thread only every 1 millisecond certainly would decrease performance pretty badly. Perhaps sleep for 20 or even 50 seconds?